### PR TITLE
Rest api#3

### PR DESCRIPTION
**Pull request by SergeiKrutii**
по заданию №3

### CODE REVIEW

File: controllers/contacts.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+const Joi = require("joi");
+const { HttpError, ctrlWrapper } = require("../helpers");
+const { Schema, model } = require("mongoose");
+
+const contactsSchema = new Schema({
```
Comment (DmytroGupanets) [inline controllers/contacts.js]: Move Model Schema into models => contacts

Comment (DmytroGupanets) [reply controllers/contacts.js]: Also, in this pull request or the next one, create a "contacts" folder in the "controllers" directory, where all the controllers handling "/contact" requests will reside. Divide all the controllers into files: addContact.js, updateContact.js, etc. Add a default index.js file for re-exporting the controllers.

**Review by DmytroGupanets — CHANGES_REQUESTED** [review]
Read comments (2)

File: controllers/contacts.js (hunk 1 above)
Comment (SergeiKrutii) [reply controllers/contacts.js]: i think its done, check plaese

**Review by DmytroGupanets — APPROVED** [review]
LGTM

**Pull request merged**